Added Assets content to What Does OCS Do (#367)
osisoft/OCS-Docs: Content_Portal/Overview/what-does-ocs-do.md

OCS provides a graphical interface for setting up stream views or you can use REST APIs to define stream views programmatically. If you are using the .NET framework, OSIsoft also offers client libraries to help you create and use stream views. 

## Data analysis 

After defining types, streams and stream views as needed, you can use the analytical tools provided by OCS to sort and visualize data from these objects. 

Two analytical tools are available in OCS: 

* [Trend](#trend) 
* [Assets](#assets) 

### Trend

The Trend feature converts stream data to a graphic view, which can reveal trends, high points, or trouble spots. Use Trend to select data streams in a namespace, specify a time range, and then render a graph of those data values.

The following example shows stream data for two streams over a selected period of several days:

![OCS](images/how-does-ocs-work/trend.jpg)

### Assets

> Assets are a framework that enables you to model data for different groupings of physical entities. An asset can consist of any number of data streams, and can be thought of as a digital twin for any physical entity. 

Reviewer: ```suggestion
Assets are a digital twin .... that enables you to model data for different groupings of physical entities. An asset can consist of any number of data streams, and can be thought of as a digital twin for any physical entity. 
```

Author: Done.

> The assets in the example below each represent trucks in a fleet, with multiple data streams defined for each.  Any selected vehicle asset can be used to display  visualization of selected data streams for review and analysis, as in the following example:

Reviewer: ```suggestion
The assets in the example below represent trucks in a fleet, with multiple data streams defined for each.  You can select an asset and display a trend of the data streams which can be used to identify and analyze problems.
```

![OCS](images/how-does-ocs-work/assets-example.png)

> 1. Select each of a truck's four suspension cylinders to display visualization of each cylinder's variation in pressure over time.

Reviewer: ```suggestion
You can display the variation in the pressure of the suspension cylinders on each truck over time.
```

> 2. Identify cylinders whose pressure is outside the accepted range to troubleshoot suspension issues.

Reviewer: Remove this sentence.

Author: Right - - Done.

> Assets are a great way to organize and contextualize data streams.  With PI to OCS transfers, for example, they provide a means of organizing multiple PI data tags on the OCS side into multiple streams under a single asset. For example, you could model an arena with streams measuring data for thermostats, ventilation equipment, lighting systems, and security as an asset based on a single entity with streams for multiple devices.

Reviewer: ```suggestion
Assets are a useful way to organize and contextualize data streams.  With PI to OCS data transfers, for example, you can organize multiple PI tags under a single asset. For example, you could create an asset with streams measuring data for thermostats, ventilation equipment, lighting systems, and security.
```

> To define an asset, you can define an asset type to reuse for other assets, just as you would define a stream type for a stream.  While for stream types, the properties you define are data fields in a stream, the properties you define for asset types are streams to be included in an asset. However, assets can also be non-typed, and created on an ad hoc basis.

Reviewer: ```suggestion
Assets can be created on an ad hoc basis or they can be created from an asset type, a template for creating similar assets. 
```

> Users can also define asset rules, which bulk create or update assets according to matching patterns in stream names and metadata.  Asset rules are applied to existing applicable streams, and whenever a stream is created, updated, or deleted.

Reviewer: Mark, I am surprised Victor left this in. This feature is not yet in the product. It is part of the M4 release. I would remove this or at least verify with him that this feature is being exposed.

Author: Let me check with him....

Author: Hi Laureen,
I just emailed you with Victor's response.

Reviewer: Please add the caveat that this can be done using the API.

Author: Done
